client/packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -14,6 +14,8 @@ export interface CellProps<T extends RecordWithId> {
   isDisabled?: boolean;
   // Unique name for browser autocomplete (to remember previously entered values for that name)
   autocompleteName?: string;
+  t: TypedTFunction<LocaleKey>;
+  d: (date: string | number | Date) => string;
```

Comment:
why the short var names? maybe a comment for `t` which is not obvious to me...

Reply:
yeah, was just taking the `t` from elsewhere. it is used throughout, to I just ran with it. I'll update.
am looking into virtualisation techniques, will see if I can improve